fda-model-test/unittests/hydraulics/HydraulicDatasetShould.cs

```diff
@@ -5,12 +5,32 @@
 using HEC.FDA.Model.structures;
 using System.Linq;
 using HEC.FDA.Model.hydraulics.Interfaces;
+using HEC.FDA.Model.paireddata;
 
 namespace HEC.FDA.ModelTest.unittests.hydraulics
 {
     [Trait("RunsOn", "Remote")]
+    [Collection("Serial")]
     public class HydraulicDatasetShould
     {
+        private const string ParentDirectoryToSteadyResult = @"..\..\..\fda-model-test\Resources\MuncieSteadyResult";
+        private const string SteadyHDFFileName = @"Muncie.p10.hdf";
+        private const string PathToIndexPointShapefile = @"..\..\..\fda-model-test\Resources\MuncieIndexPoints\MuncieIndexPts.shp";
+        [Fact]
+        void retreiveGraphicalFrequencyFunctionsAsPairedData()
+        {
+            //Arrange
+            List<IHydraulicProfile> profiles = new List<IHydraulicProfile>();
+            profiles.Add( new HydraulicProfile(0.50, SteadyHDFFileName, "2"));
+            profiles.Add(new HydraulicProfile(0.002, SteadyHDFFileName, "500"));
+            HydraulicDataset dataset = new HydraulicDataset(profiles, HydraulicDataSource.SteadyHDF);
+
+            //Act
+            List<UncertainPairedData> graphicalFreqCurves = dataset.GetGraphicalStageFrequency(PathToIndexPointShapefile, ParentDirectoryToSteadyResult);
+
+            //Assert
+            System.Console.WriteLine("poo");
```

Comment:
Ha! Looks like we could use a TODO statement here on the assertion

Reply:
whoops. This is pretty tame for my nonsense writelines. Got lucky there. Good catch.

Reply:
ammended